doc/examples/gestures/lib/main_multitap_advanced.dart

```diff
@@ -49,37 +50,31 @@ class MyGame extends BaseGame
   }
 
   @override
-  void onReceiveDrag(DragEvent event) {
-    onPanStart(event.initialPosition);
-
-    event
-      ..onUpdate = onPanUpdate
-      ..onEnd = onPanEnd
-      ..onCancel = onPanCancel;
-  }
-
-  void onPanCancel() {
+  void onDragCanceled(int pointerId) {
     _end = null;
     _start = null;
     _panRect = null;
   }
 
-  void onPanStart(Offset position) {
+  @override
+  void onDragStarted(int pointerId, Vector2 initialPosition) {
```

Comment:
Isn't it better to name it `onDragStart` to follow the Flutter gesture convention with `onPanStart` etc? And the same for the other methods.